libcxx/test/std/utilities/expected/expected.void/observers/value.lwg3940.verify.cpp

```diff
@@ -0,0 +1,27 @@
+//===----------------------------------------------------------------------===//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+// ADDITIONAL_COMPILE_FLAGS: -Xclang -verify-ignore-unexpected=error
```

Comment:
Why did you add this line?

Reply:
Sorry I just copy from example file in https://github.com/llvm/llvm-project/pull/71819#discussion_r1394171789, and forget to delete this flag.

Now it's deleted.